new-integration-tests/src/test/java/oracle/weblogic/domain/ClusterStatus.java

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.domain;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+@ApiModel(description = "ClusterStatus describes the current status of a specific WebLogic cluster.")
+public class ClusterStatus {
+
+  @ApiModelProperty("WebLogic cluster name. Required.")
+  private String clusterName;
+
+  @ApiModelProperty(value = "The number of intended cluster members. Required.", allowableValues = "range[0,infinity]")
+  private Integer replicas;
+
+  @ApiModelProperty(value = "The number of ready cluster members. Required.", allowableValues = "range[0,infinity]")
+  private Integer readyReplicas;
+
+  @ApiModelProperty(value = "The maximum number of cluster members. Required.", allowableValues = "range[0,infinity]")
+  private Integer maximumReplicas;
+
+  @ApiModelProperty(value = "The minimum number of cluster members.", allowableValues = "range[0,infinity]")
+  private Integer minimumReplicas;
+
+  @ApiModelProperty(value = "The requested number of cluster members from the domain spec. "
+      + "Cluster members will be started by the operator if this value is larger than zero.",
+      allowableValues = "range[0,infinity]")
+  private Integer replicasGoal;
+
+  public ClusterStatus clusterName(String clusterName) {
+    this.clusterName = clusterName;
+    return this;
+  }
+
+  public String getClusterName() {
+    return clusterName;
+  }
+
+  public void setClusterName(String clusterName) {
+    this.clusterName = clusterName;
+  }
+
+  public ClusterStatus replicas(Integer replicas) {
+    this.replicas = replicas;
+    return this;
+  }
+
+  public Integer getReplicas() {
+    return replicas;
+  }
+
+  public void setReplicas(Integer replicas) {
+    this.replicas = replicas;
+  }
+
+  public ClusterStatus readyReplicas(Integer readyReplicas) {
```

Comment:
Do you want to use fluent API or not?  readyReplicas is, but setReplicas is not.  Please explain, maybe this is intended, and I am just confused.

Reply:
The types had both, so I kept them.  I'm happy to delete set*